### PR TITLE
Updated Fullcalendar, fixed list's popover, minor corrections.

### PR DESCRIPTION
**Pull request by poradz**
Updated Fullcalendar, fixed list's popover, minor corrections.

### CODE REVIEW

File: public_html/layouts/basic/modules/Calendar/resources/CalendarView.js [hunk 1]
```diff
@@ -164,19 +164,22 @@ jQuery.Class("Calendar_CalendarView_Js", {
...
-						cb: callbackFunction
-					};
-					app.showModalWindow(modalWindowParams);
-				});
+
+			},
+			eventClick: function (calEvent, jsEvent, view) {
+				jsEvent.preventDefault();
+				var linkSplit = $(this).context.href.split("=");
+				var recordNumber = linkSplit[linkSplit.length - 1];
```
Comment (mariuszkrzaczkowski) [inline public_html/layouts/basic/modules/Calendar/resources/CalendarView.js]: there is a function of parsing this

File: public_html/layouts/basic/modules/OSSTimeControl/resources/Calendar.js [hunk 2]
```diff
@@ -127,10 +127,6 @@ jQuery.Class("OSSTimeControl_Calendar_Js", {
 							(event.subprocl ? '<div><span class="userIcon-' + event.subprocm + '" aria-hidden="true"></span> <label>' + app.vtranslate('JS_SUB_PROCESS') + '</label>: <a target="_blank" href="index.php?module=' + event.subprocm + '&view=Detail&record=' + event.subprocess + '">' + event.subprocl + '</a></div>' : '') +
 							(event.smownerid ? '<div><span class="glyphicon glyphicon-user" aria-hidden="true"></span> <label>' + app.vtranslate('JS_ASSIGNED_TO') + '</label>: ' + event.smownerid + '</div>' : '')
 				});
-				element.find('.fc-content, .fc-info').click(function () {
```
Comment (mariuszkrzaczkowski) [inline public_html/layouts/basic/modules/OSSTimeControl/resources/Calendar.js]: why it has been removed

File: public_html/layouts/basic/modules/Reservations/resources/Calendar.js [hunk 3]
```diff
@@ -125,10 +125,6 @@ jQuery.Class("Reservations_Calendar_Js", {
 							(event.process ? '<div><span class="userIcon-' + event.processType + '" aria-hidden="true"></span> <label>' + event.processLabel + '</label>: <a target="_blank" href="index.php?module=' + event.processType + '&view=Detail&record=' + event.processId + '">' + event.process + '</a></div>' : '') +
 							(event.smownerid ? '<div><span class="glyphicon glyphicon-user" aria-hidden="true"></span> <label>' + app.vtranslate('JS_ASSIGNED_TO') + '</label>: ' + event.smownerid + '</div>' : '')
 				});
-				element.find('.fc-content, .fc-info').click(function () {
```
Comment (mariuszkrzaczkowski) [inline public_html/layouts/basic/modules/Reservations/resources/Calendar.js]: why it has been removed

File: public_html/layouts/basic/modules/Vtiger/resources/DetailPreview.js [hunk 4]
```diff
@@ -12,15 +15,15 @@ Vtiger_Detail_Js("Vtiger_DetailPreview_Js", {}, {
...
+	 * Redirects to the current iframe's parent.
 	 */
 	updateParentFrame: function () {
 		parent.app.getPageController().updateWindowHeight($(".mainContainer").height(), $(window.frameElement));
 	},
 	/**
-	 * Function sets correct size of iframes
-	 * 
+	 * Function sets the correct iframe's size.
+	 * @param {$} currentHeight - ifrmae's body height to be set.
```
Comment (mariuszkrzaczkowski) [inline public_html/layouts/basic/modules/Vtiger/resources/DetailPreview.js]: {$}

File: public_html/layouts/basic/modules/OSSTimeControl/resources/Calendar.js (hunk 2 above)
Comment (poradz) [reply public_html/layouts/basic/modules/OSSTimeControl/resources/Calendar.js]: This event needed changes in the library to work. I exchange it with the bulit-in library "eventClick".

File: public_html/layouts/basic/modules/Vtiger/resources/DetailPreview.js (hunk 4 above)
Comment (poradz) [reply public_html/layouts/basic/modules/Vtiger/resources/DetailPreview.js]: I decided to use $ for jquery objects, what do you think about that?

File: public_html/layouts/basic/modules/OSSTimeControl/resources/Calendar.js (hunk 2 above)
Comment (poradz) [reply public_html/layouts/basic/modules/OSSTimeControl/resources/Calendar.js]: This event was needed becose of our changes in the library. Now clicked record automatically redirects.

File: public_html/layouts/basic/modules/Reservations/resources/Calendar.js (hunk 3 above)
Comment (poradz) [reply public_html/layouts/basic/modules/Reservations/resources/Calendar.js]: The same as above.

File: public_html/layouts/basic/modules/Vtiger/resources/DetailPreview.js [hunk 5]
```diff
@@ -36,30 +39,18 @@ Vtiger_Detail_Js("Vtiger_DetailPreview_Js", {}, {
...
-			thisInstance.updateParentFrame();
-		});
-		app.event.on('DetailView.Tab.AfterLoad', function () {
-			thisInstance.updateParentFrame();
-		});
-		app.event.on("DetailView.SaveComment.AfterLoad DetailView.SaveComment.AfterUpdate", function () {
-			thisInstance.updateParentFrame();
-		});
-		app.event.on("DetailView.Widget.AfterLoad DetailView.UpdatesWidget.AddMore", function () {
+		var wrapper = $('.mainContainer ');
```
Comment (mariuszkrzaczkowski) [inline public_html/layouts/basic/modules/Vtiger/resources/DetailPreview.js]: ?

**Pull request merged**